polkadot/node/core/runtime-api/src/lib.rs

```diff
@@ -531,6 +537,12 @@ where
 			ver = 1,
 			sender
 		),
+		Request::CandidatesPendingAvailability(para, sender) => query!(
+			CandidatesPendingAvailability,
+			candidates_pending_availability(para),
+			ver = 1,
```

Comment:
This should be 11.

Add a new const [here](https://github.com/paritytech/polkadot-sdk/pull/4027/files#diff-e4ba49b765a03768afe33710f5056b84a6e069775d58945771d450d3efc638fdR782):
```rust
/// `ClaimQueue`
	pub const CANDIDATE_PENDING_AVAILABILITY_REQUIREMENT: u32 = 11;
```
and then it can be used for version checks too.

Reply:
Nice catch!